sdk/all/src/main/scala/org/typelevel/otel4s/sdk/OpenTelemetrySdk.scala

```diff
@@ -322,7 +323,7 @@ object OpenTelemetrySdk {
 
     /** Creates a [[Builder]].
       */
-    def builder[F[_]: Async: Parallel: Env: SystemProperties: Console: LocalContextProvider]: Builder[F] =
+    def builder[F[_]: Async: Parallel: Env: SystemProperties: Console: Diagnostic: LocalContextProvider]: Builder[F] =
```

Comment:
Do we still need `Console` here?

Reply:
Unfortunately, yes. We need `Console` in `Console*` exporters, for example [ConsoleSpanExporter](https://github.com/typelevel/otel4s/blob/2a21fca9452aa173720c675f369f24a83b7ebb33/sdk/trace/src/main/scala/org/typelevel/otel4s/sdk/trace/exporter/ConsoleSpanExporter.scala).